### PR TITLE
Tijlleenders main

### PR DESCRIPTION
**Pull request by GithinjiHans**

### CODE REVIEW

**Comment by tijlleenders** [comment]
Should the package-lock.json be deleted? 
[This article from a relatively authoratative source says no](https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5).

**Comment by Tushar-4781** [comment]
The article is correct, it shouldn't be deleted. Also, Cypress will fail to run the test.

**Comment by Tushar-4781** [comment]
@GithinjiHans, please make sure to give informative names to the pull requests. The branch name is not the convention to follow for PR naming.

File: src/components/todoList/TodoListForm.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React,{useState} from 'react'
+import { Button } from 'react-bootstrap';
+import { useRecoilValue } from 'recoil';
+import { darkModeState } from '../../store/DarkModeState'
+import "./TodoList.scss";
+import AddIconLight from "../../images/AddIconLight.png";
+import AddIconDark from "../../images/AddIconDark.png";
+
+
```
Comment (Tushar-4781) [inline src/components/todoList/TodoListForm.jsx:9]: Don't add more than one line gap between the code.

File: src/translations/hi/translation.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "explore": "अन्वेषण",
     "zinzen": "ज़िनज़ेन",
     "feelingsmessage": "नमस्ते! तुम्हें आज कैसा लग रहा हा?",
+    "myGoalsMessage":"",
```
Comment (Tushar-4781) [inline src/translations/hi/translation.json:17]: Maybe use a translation service like google translate.

**Review by Tushar-4781 — APPROVED** [review]

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
Bring back the package-lock.json.

File: src/translations/hi/translation.json (hunk 2 above)
Comment (GithinjiHans) [reply src/translations/hi/translation.json:17]: Done

**Pull request closed without merging**